.rubocop.yml

```diff
@@ -1596,19 +1593,17 @@ Style/NestedTernaryOperator:
 Style/Next:
   Enabled: false
 
-# Disabling for now
 Style/NilComparison:
-  Enabled: false
-  # EnforcedStyle: predicate
+  Enabled: true
+  EnforcedStyle: predicate
```

Comment:
This cop's config is now redundant with [these defaults](https://github.com/rubocop/rubocop/blob/master/config/default.yml#L4366). We can remove all of these lines from .rubocop.yml

Reply:
Nice catch! Resolved by: 347dcbc919d0c1d5ad64891dee9e39346af654aa